examples/Authentication/google_ads_php.ini

```diff
@@ -46,3 +46,7 @@ refreshToken = "INSERT_OAUTH2_REFRESH_TOKEN_HERE"
 ; Optional transport settings.
 ; By default, "grpc" is used if available otherwise "rest".
 ; transport = "grpc"
+
+; Optional gRPC transport settings.
+; By default, false is used.
```

Comment:
Please add more details on this. For a person with no background about insecure channel, it's hard to understand when and how we can use this option and what the consequence it will have.